### ISSUE TITLE
missing tooltip role since prime 20

### ISSUE THREAD

**Issue by makepanic** [primefaces/primeng#19048]
### Describe the bug

In primeng@19 the container received a `role="tooltip"`. 
https://github.com/primefaces/primeng/blob/v19-prod/packages/primeng/src/tooltip/tooltip.ts#L397

This behavior seemed to be removed in primeng@20. This feels like a a11y regression to me.
https://github.com/primefaces/primeng/blob/master/packages/primeng/src/tooltip/tooltip.ts#L403-L438

### Pull Request Link

https://github.com/primefaces/primeng/pull/19046

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-nulp6n9e-9g1vd231?file=src%2Fapp%2Fapp.component.html

### Environment

Any primeng@20 angular installation.

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Hover a tooltip
2. Check the tooltip dom
3. See no role="tooltip"

<img width="1432" height="96" alt="Image" src="https://github.com/user-attachments/assets/1cb91993-1489-4f11-a95e-287472d0d176" />

### Expected behavior

1. Hover a tooltip
2. Check the tooltip dom
3. See role="tooltip"

<img width="1432" height="96" alt="Image" src="https://github.com/user-attachments/assets/6ef25c5e-84f5-441a-a7c2-b9aadcf84bf0" />

**Comment by cetincakiroglu** [comment]
Hi,

You can use the new passthrough feature to provide role or any attribute you want.

```
<input type="text" pInputText pTooltip="Enter your username" tooltipPosition="right" placeholder="Right" [ptTooltip]="{ root: { role: 'ROLE' } }" />

```

**Comment by makepanic** [comment]
Just wondering does that mean primeng components are expected to not be accessible by default?
